.travis.yml

```diff
@@ -40,6 +40,10 @@ addons:
       - gcc-multilib
       - lib32gcc-6-dev
 
+before_script:
+  - wget http://releases.linaro.org/components/toolchain/binaries/6.3-2017.02/aarch64-linux-gnu/gcc-linaro-6.3.1-2017.02-x86_64_aarch64-linux-gnu.tar.xz
```

Comment:
Well, it doubles the amount downloaded and increases the run time from ~4 mins to ~6 mins..

Maybe this would be better suited to run in cbuild? Someone can run the cbuild tests during the release cycle..